composer.json

```diff
@@ -62,8 +62,8 @@
     "etdsolutions/waypoints": "4.0.0",
     "twig/twig": "2.13.*",
     "twig/extensions": "1.5.*",
-    "twitter/bootstrap": "3.3.*",
-    "twitter/typeahead.js": "v0.10.5",
+    "twbs/bootstrap": "5.0.*",
+    "twitter/typeahead.js": "v0.11.*",
```

Comment:
Note to self: before making it a final PR, confirm whether there's a new version of twbs/bootstrap and update here if necessary.